content/en/docs/concepts/workloads/controllers/_index.md

```diff
@@ -3,3 +3,41 @@ title: "Workload Resources"
 weight: 20
 ---
 
+Kubernetes provides several built-in APIs for declarative management of your
+{{< glossary_tooltip text="workloads" term_id="workload" >}}
+and the components of those workloads.
+
+You use Kubernetes API to create {{< glossary_tooltip text="object" term_id="object" >}}
+(HTTP resources) that represent these higher level abstractions, and then let the
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}} automatically manage
+Pods on your behalf, based on the specification for the workload object you defined.
+
+The built-in APIs for managing workloads are:
+
+[`Deployment`](/docs/concepts/workloads/controllers/deployment/) and [`ReplicaSet`](/docs/concepts/workloads/controllers/replicaset/)
+(replacing the legacy resource
+{{< glossary_tooltip text="ReplicationController" term_id="replication-controller" >}}).
+`Deployment` is a good fit for managing a stateless application workload on your cluster,
+where any {{< glossary_tooltip term_id="Pod" >}} in the `Deployment` is interchangeable
+and can be replaced if needed.
+
+A [`StatefulSet`](/docs/concepts/workloads/controllers/statefulset/) lets you
+manage one or more Pods – all running the same application code – that do track state somehow.
+For example, if your workload records data persistently, you can run a `StatefulSet` that
+matches each `Pod` with a [`PersistentVolume`](/docs/concepts/storage/persistent-volumes/).
+Your code, running in the `Pods` for that `StatefulSet`, can replicate data to
```

Comment:
We say `pods` as a resource collection, or `Pod` as a resource type, not `Pods`.

```suggestion
Your code, running in the `Pod`s for that `StatefulSet`, can replicate data to
```

Reply:
I'm frustrated to see criticism of existing text. This wording has been [reviewed](https://github.com/kubernetes/website/pull/24602#issuecomment-748408719) once already and all I'm doing here is _moving_ it.

https://github.com/kubernetes/website/blame/0b0c6a813abb69e234618d671bc9360631c82dbe/content/en/docs/concepts/workloads/_index.md#L38 shows how this looks on the `main` branch before the PR.

Reply:
There is no hint showing that these texts are **moved**. In this PR, the whole file is rendered as **new text**.
I'm reviewing the text as is.

Speaking of frustration, I'm feeling the same thing when seeing text shuffled here and there for personal tastes rather than any good reasons. This kind of "improvement" (as refactoring again) is introducing a lot of work for the downstream localization teams. Those people deserve some respects as well.

Reply:
GitHub doesn't provide a brilliant way to spot when text is moved between files. Sorry about that. It doesn't help much for this particular case but `git diff --color-moved` often does let you see what has moved.

The motivation for the move is because someone totally new to Kubernetes might see https://kubernetes.io/docs/concepts/workloads/controllers/ and section overviews like this, where there's no overview at all.

After a few of these, especially if you've been reading for half an hour, you might consider giving up. I still remember that new-user frustration; it's one of the main reasons I started contributing to the docs and why I still do.

Reply:
I'll mark this resolved.